ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -747,6 +748,8 @@ std::tuple<ttnn::Tensor, uint32_t, uint32_t, ttnn::Tensor, std::optional<ttnn::T
     auto opt_conv_op_parallel_config = determine_conv_op_parallel_config_from_conv_output_mem_config(
         conv_out_memory_config, get_num_cores_nhw_from_parallel_config(parallel_config),
         get_num_cores_channels_from_parallel_config(parallel_config));
+
+    if(conv_config.shard_layout == TensorMemoryLayout::WIDTH_SHARDED) {  conv_config.act_block_h_override = 0; }
```

Comment:
didn't we agree that we shouldn't do this and that instead code reading from act_block_h_override should take into accoutn that it's dealing with width sharding?

Reply:
I misunderstood initially. I have fixed it.